wfe/web-front-end.go

```diff
@@ -975,25 +974,22 @@ func (wfe *WebFrontEndImpl) Authorization(response http.ResponseWriter, request
 	}
 
 	// Blank out ID and regID
-	switch request.Method {
-	case "GET":
```

Comment:
This code was actually meaningful and tests should be failing on it because POSTs have meaning for this endpoint but are just handled above this code. Please undo this change and I think maybe we need to scale back this diff completely.

This is a lot of churn.